tests/scattering3d/test_torch_scattering3d.py

```diff
@@ -85,54 +95,31 @@ def test_cdgmm3d(device, backend, inplace):
         if inplace:
             assert (x - z).norm().cpu().item() < 1e-7
 
-        with pytest.warns(UserWarning) as record:
```

Comment:
Aren't all these tested in the base already? Why have 3D-specific tests?

Reply:
Good question. I believe when I started this refactor the base test wasnt there. Lemme look through the base test.

Reply:
It doesn't seem like they're tested in the general tests.

Reply:
Do we wanna do this in a later pr?

Reply:
I can put an issue up.

Reply:
Sounds good. Please do.